CRM/Booking/Form/Booking/Info.php

```diff
@@ -406,15 +406,25 @@ public static function registerScripts() {
     }
     $loaded = TRUE;
 
+    // Based on from CRM_UF_Page_ProfileEditor::registerProfileScripts()
+    Civi::resources()
+      ->addScriptFile('civicrm.packages', 'backbone/json2.js', 100, 'html-header', FALSE)
+      ->addScriptFile('civicrm.packages', 'backbone/backbone.js', 120, 'html-header')
+      ->addScriptFile('civicrm.packages', 'backbone/backbone.marionette.js', 125, 'html-header', FALSE)
+      ->addScriptFile('civicrm.packages', 'backbone/backbone.collectionsubset.js', 125, 'html-header', FALSE)
+      ->addScriptFile('civicrm.packages', 'backbone-forms/distribution/backbone-forms.js', 130, 'html-header', FALSE)
+      ->addScriptFile('civicrm.packages', 'backbone-forms/distribution/adapters/backbone.bootstrap-modal.min.js', 140, 'html-header', FALSE)
+      ->addScriptFile('civicrm.packages', 'backbone-forms/distribution/editors/list.min.js', 140, 'html-header', FALSE)
+      ->addStyleFile('civicrm.packages', 'backbone-forms/distribution/templates/default.css', 140, 'html-header')
+      ->addScript('CRM.BB = Backbone.noConflict();', 300, 'html-header')
+      ->addStyleFile('civicrm', 'css/crm.designer.css', 140, 'html-header')
+      ->addScriptFile('civicrm', 'js/crm.backbone.js', 150);
+
     CRM_Core_Resources::singleton()
```

Comment:
Kinda weird to mix the new and old way of referring to `Civi::resources()` a few lines apart.